editor/src/clj/editor/search_results_view.clj

```diff
@@ -307,3 +328,267 @@
+   :children (into []
+                   (map-indexed
+                     (fn [i v]
+                       {:fx/type fxui/label
+                        :grid-pane/column i
+                        :grid-pane/halignment :right
+                        :style-class (overridden-style-classes property)
+                        :text (field-expression/format-number v)}))
+                   value)})
+
+(defmethod override-value-cell-view types/Vec2 [property]
+  (multi-text-field property))
+
+(defmethod override-value-cell-view types/Vec3 [property]
+  (multi-text-field property))
+
+(defmethod override-value-cell-view types/Vec4 [property]
+  (multi-text-field property))
+
+(defmethod override-value-cell-view types/Color [{:keys [value] :as property}]
+  (if value
+    (let [[r g b a] value
+          color (Color. r g b a)
+          nr (Math/round (double (* 255 r)))
+          ng (Math/round (double (* 255 g)))
+          nb (Math/round (double (* 255 b)))
+          na (Math/round (double (* 255 a)))]
+      {:fx/type fxui/label
+       :style-class (overridden-style-classes property)
+       :graphic {:fx/type fx.region/lifecycle
+                 :min-width 10
+                 :min-height 10
+                 :background {:fills [{:fill color}]}}
+       :text (if (= 255 na)
+               (format "#%02x%02x%02x" nr ng nb)
+               (format "#%02x%02x%02x%02x" nr ng nb na))})
+    {:fx/type fxui/label}))
+
+(defmethod override-value-cell-view :choicebox [{:keys [edit-type value] :as property}]
+  (let [labels (into {} (:options edit-type))]
+    {:fx/type fxui/label
+     :style-class (overridden-style-classes property)
+     :text (get labels value)}))
+
+(defn open-hyperlink-resource! [open-resource-fn resource _]
+  (open-resource-fn resource {}))
+
+(defmethod override-value-cell-view resource/Resource [{:keys [value open-resource-fn] :as property}]
+  {:fx/type fx.hyperlink/lifecycle
+   :style-class (into ["override-inspector-hyperlink"] (overridden-style-classes property))
+   :on-action (fxui/partial #'open-hyperlink-resource! open-resource-fn value)
+   :text (resource/resource->proj-path value)})
+
+(defn- value-cell [open-resource-fn property]
+  {:graphic (override-value-cell-view (assoc property :value (properties/value property)
+                                                      :open-resource-fn open-resource-fn))})
+
+(defn- property-value [property-keyword tree]
+  (-> tree :properties property-keyword))
+
+(defn- tree-table-view-event-filter [open-resource-fn ^Event e]
+  (when (and (instance? MouseEvent e)
+             (= MouseEvent/MOUSE_PRESSED (.getEventType e)))
+    (let [^MouseEvent e e]
+      (when (even? (.getClickCount e))
+        (.consume e)
+        (when-let [tree (-> e
+                            ^TreeTableView .getSource
+                            .getSelectionModel
+                            ^TreeItem .getSelectedItem
+                            (some-> .getValue))]
```

Comment:
In the past, we've had problems with tagging inside the threading macros - it was not being applied correctly in release builds and resulted in reflection. Maybe consider breaking this out into a separate function to avoid that?